keep/providers/grafana_provider/grafana_provider.py

```diff
@@ -268,10 +268,10 @@ def _format_alert(
                 description=description,
                 source=["grafana"],
                 labels=labels,
-                url=url,
-                imageUrl=image_url,
-                dashboardUrl=dashboard_url,
-                panelUrl=panel_url,
+                url=url or None,
+                imageUrl=image_url or None,
+                dashboardUrl=dashboard_url or None,
+                panelUrl=panel_url or None,
```

Comment:
mmm, when would that be affected?

if url is string/int/whatever it'll be "stronger" than None

Reply:
An empty string is not stronger. And if it is not empty - then it will be really invalid.

Reply:
🆗

Reply:
```
>>> ('' or None) is None
True
```